project-reports/2024/2024-annual-Hyperledger-Indy.md

```diff
@@ -0,0 +1,98 @@
+
+As this is the first Hyperledger Indy Annual Report, there is not a 2022 Annual Report to look back on. However, in March 2023, the Indy Contributors group developed this [Indy Roadmap](https://hackmd.io/GeRP00i0Sj-7z4zXn2MB5g?view), an (non-timelined) wish list of technical capabilities we’d like to see in Indy. That roadmap was part of the agenda of the Hyperledger Indy Ecosystem summit, where it was highlighted that in addition to the technical capabilities, a focus was needed on marketing and on influencing regulatory guidance on decentralized identity infrastructure.
+
+Of those items, progress was made on the following items:
+
+* Ubuntu 20.04 development, deployment, upgrade documentation, and release.
+* Clean up of issues in the various Indy repositories.
+* Indy node monitoring capabilities.
+
+That is less progress than we would have liked to see. A positive spin on that lack of progress is that Indy continues to be stable and adequately scalable such that there is no need 
+
+There were several events around which Hyperledger Indy code development efforts were focused over the year:
+
+* Completion of the Ubuntu 20.04 version of Indy Node. Deployers of Indy networks have reported success in deploying the new version.
+* The archiving of Ursa, triggering the move of the BLS Signatures library from Ursa to being maintained in the Indy Project.
+* The movement of Aries projects to use the “shared components” in place of the Indy SDK. The “shared components” consist of [Aries Askar](https://github.com/hyperledger/aries-askar) (secure storage), [Indy VDR](https://github.com/hyperledger/indy-vdr) (client interface to Indy networks) and [Hyperledger AnonCreds](https://www.hyperledger.org/projects/anoncreds) (AnonCreds implementation).
+* The deprecation of the Indy SDK.
+
+In addition, as mentioned earlier in the report, significant progress was made in creating an Indy-Besu proof of concept.
+
+### Next Year’s Goals
+
+_Guidance: What are the goals for the next year of the project? The goals should list what you want to achieve, not just what you know you can achieve. Feel free to include stretch goals and things that you are looking to explore in the next year. For example, are you working on major new features? Or are you concentrating on adoption, community growth, or documentation?_
+
+The goals for 2024 derive from both 2023 Indy Roadmap document, and the outcome of the Indy Ecosystems Summits held in 2023. The bullet point versions of these goals include the following. The Indy maintainers and contributors have a deeper understanding of what is behind these bullet points:
```

Comment:
Indy-Besu goals? Or are we considering a separate work bucket?

Reply:
Doh—good point, we should have had it in there. Added and noted that some of the goals might be accomplished via Indy Node or in Indy Besu.